src/displayIcon/DisplayIcon.styles.tsx

```diff
@@ -1,3 +1,4 @@
+// @ts-nocheck
```

Comment:
@felipesoares6 I see that you disable  type check for style files. But should we create proper types for variant theme and backgroundColor? 

something like this? And this type can stay in common folder or somewhere generic so that whenever we use theme or variant we can use these types ?

```
background-color: ${({ theme, variant }: { theme: Theme; variant: Variant }) =>
    (backgroundColors as BackgroundColors)[theme.c7__ui.mode][variant]};
```

Reply:
I'm ignoring most of the style types for the first iteration because we might need to update the styled-components library soon. With this next upgrade, the library will provide the types for the theme "automatically" since the next version also uses TypeScript so it will be easier to type it.

Also, it's just more work to take care of this right now too, since we are migrating 49 components at once 😢